asko_import/importer.py

```diff
@@ -429,14 +428,6 @@ def _validate_imported_data():
         with log_context_from(reservation):
             LOG.error("Reservation does not have an application")
 
-    LOG.info("Checking that %s", "queue position 1 is not submitted...")
-    in_bad_state_with_queue_pos_1 = reservations.filter(
-        queue_position=1, state=ApartmentReservationState.SUBMITTED
-    )
-    for reservation in in_bad_state_with_queue_pos_1:
-        with log_context_from(reservation):
-            LOG.error("Reservation in queue pos 1 is submitted")
-
```

Comment:
Restore this and add code that in the end of everything changes the SUBMITTED on position 1 to RESERVED.